src/main/java/seedu/address/model/person/Email.java

```diff
@@ -13,18 +13,24 @@
     public static final String MESSAGE_EMAIL_CONSTRAINTS =
             "Person emails should be 2 alphanumeric/period strings separated by '@'";
     public static final String EMAIL_VALIDATION_REGEX = "[\\w\\.]+@[\\w\\.]+";
-
     public final String value;
 
     /**
+     * The default Email constructor when email is not specified by the user
+     */
+    public Email(){
+        value = "<Unspecified email>";
+    }
+
+    /**
      * Validates given email.
-     *
+     * An exception to the ILLegalValueException is the String "Unspecified email"
      * @throws IllegalValueException if given email address string is invalid.
      */
     public Email(String email) throws IllegalValueException {
         requireNonNull(email);
         String trimmedEmail = email.trim();
-        if (!isValidEmail(trimmedEmail)) {
+        if (!isValidEmail(trimmedEmail) && !trimmedEmail.equals("<Unspecified email>")) {
```

Comment:
Should check for equality with the unspecified object.

Reply:
Ive removed it completely since I implemented it slightly differently now